# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/README_MOZILLA
# Commit: 32f7037461f2
# Full Hash: 32f7037461f260c69b8fb9893c3345c672210144
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-13 21:06:53
# Description:
#   Bug 1879740 - Don't pass an UTF-8 string to system vsnprintf. r=media-playback-reviewers,chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D201606
# ==============================================================================

diff -r c5246b9e5ee6 -r 32f7037461f2 media/ffvpx/README_MOZILLA
--- a/media/ffvpx/README_MOZILLA	Tue Feb 13 16:16:00 2024 +0200
+++ b/media/ffvpx/README_MOZILLA	Tue Feb 13 14:14:31 2024 +0000
@@ -157,3 +157,7 @@
 There are going to be a lot of changes in terms of symbols exported. Adjust
 `libavutil/avutil.symbols` and `libavcodec/avcodec.symbols` by removing and
 adding symbols until the build passes.
+
+Finally, apply the patch:
+- no-unicode-stdio.patch to avoid passing the infity symbol in unicode to an
+  stdio.h function, that causes bug 1879740 issue on Windows.